build.gradle

```
@@ -0,0 +1,27 @@
+plugins {
+	id 'org.springframework.boot' version '2.6.0-SNAPSHOT'
+	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+	id 'java'
+}
+
+group = 'wolox'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '8'
```

Here we should use the constants values offer by Gradle API. Also like **optional**, you can specify the target compatibility
```suggestion
sourceCompatibility = JavaVersion.VERSION_1_8
targetCompatibility = JavaVersion.VERSION_1_8
```